_layouts/episode.html

```diff
@@ -73,7 +73,6 @@ <h2>Audio</h2>
       {{ content }}
   </div> </div> </div>
 
-  {%- include footer.html -%}
+{%- include footer.html -%}
 
-</body>
-</html>
+{%- include html-body-end.html -%}
```

Comment:
This ensures that the TCB episodes pages .. which are typically long .. also have the button